java/core/src/test/java/co/worklytics/psoxy/rules/msft/Calendar_NoAppIds_NoGroups_Tests.java

```diff
@@ -3,11 +3,12 @@
 import co.worklytics.psoxy.rules.Rules2;
 import lombok.Getter;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 //TODO: fix this re-use via inheritance; makes tests brittle; we should inject this rule set into
 // the directory tests, or something like that
-public class Calendar_NoAppIds_NoGroups_Tests extends CalendarTests {
```

Comment:
Note that all no app id tests, inherits from calendar... because they were used same set of rules + no app ids